.circleci/config.yml

```diff
@@ -368,7 +368,7 @@ jobs:
             xcrun instruments -w "iPhone 8 (13.0) [" || true
       - run:
           name: Run XCode tests
-          command: bash run-ios-tests.sh
+          command: bash automation/run_ios_tests.sh
```

Comment:
Any objection to moving this shell script out of the top-level directory @eoger @garvankeeley?